constraints.txt

```diff
@@ -0,0 +1 @@
+scipy==1.3.1
```

Comment:
This prevents `pip` from installing scipy, instead conda has to install scipy directly for it to work